src/qibolab/_core/instruments/emulator/emulator.py

```diff
@@ -39,21 +40,27 @@
 class EmulatorController(Controller):
     """Emulator controller."""
 
+    sampling_rate_: float = Field(1, exclude=True)
```

Comment:
Did you exclude for compatibility with the previous platforms? Is it required despite the default?

In case, I would just suggest not to worry too much about breaking. It is definitely better not to do it, but if that's happening rarely and in more and more isolated corners[*] it is more or less acceptable. We're not yet that stable, so it is fine to allow some wiggle room.

[*]: this not really a corner, but it would only break emulator-based platforms, at most (not even sure), and while the emulator is certainly useful, we never dedicated enough effort to claim that the overall structure is quite stable - at least, we could break something, if it's only happening seldom

Reply:
I did this in order to have the `property` and `property.setter` working below. I don't know if there is another way to make it working.

Reply:
I see, let me try to remove it to see if everything works as expected.